### DISCUSSION TITLE
posts/2023-01-15-my-research-part-3/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#55] [General]
# posts/2023-01-15-my-research-part-3/

This is the third part in a series of short articles I’m trying to write about my research. See the first one here.
In this article, I will introduce some preliminary notions that will be necessary in describing the work that I do (and in fact they are fundamental to all research in geometry). The two notions are these: manifolds and metric tensors. A manifold is a mathematical structure that generalizes and makes precise the idea of space.

https://stevenbuchanan.net/posts/2023-01-15-my-research-part-3/

**Comment by Jumitebeeninagame** [comment]
What the hell is this
